src/coreclr/src/vm/ClrEtwAll.man

```diff
@@ -3795,6 +3795,13 @@
                            task="AssemblyLoader"
                            symbol="KnownPathProbed" message="$(string.RuntimePublisher.KnownPathProbedEventMessage)"/>
 
+                    <event value="297" version="0" level="win:Informational"
+                           keywords ="GCHeapDumpKeyword"
+                           symbol="GenAwareBegin" message="$(string.RuntimePublisher.GenAwareBeginEventMessage)"/>
+
+                    <event value="298" version="0" level="win:Informational"
```

Comment:
Is there a corresponding change to TraceEvent? If so it might be good to link it from the PR.